### DISCUSSION TITLE
Question about virtual temperature

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#611] [Q&A]
@visualapproach I'm curious, what made you decide to introduce the virtual temperature determination technique rather than simply turning the pump ON for a short period to sample the actual tub water temperature and then returning the pump to the OFF state?

-Wes

**Comment by visualapproach** [comment]
To get higher resolution and/or accuracy making way for better estimates of heating times etc. The pump must run for several minutes before readings are correct. Also, people with graphing capabilities in their home assistant like to see nicer curves. And because I can ;-)
